forge-gui/res/cardsfolder/p/pongify.txt

```diff
@@ -4,4 +4,5 @@ Types:Instant
 A:SP$ Destroy | Cost$ U | ValidTgts$ Creature | TgtPrompt$ Select a creature | AILogic$ Pongify | RememberLKI$ True | AlwaysRemember$ True | SubAbility$ DBToken | NoRegen$ True | SpellDescription$ Destroy target creature. It can't be regenerated. Its controller creates a 3/3 green Ape creature token.
 SVar:DBToken:DB$ Token | TokenAmount$ 1 | TokenScript$ g_3_3_ape | TokenOwner$ RememberedController | SubAbility$ DBCleanup
 SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+DeckHas:Ability$Token & Type$Ape
```

Comment:
these ones are a bit weird... because the Ape could go to someone else

it's like Ability$Sacrifice when you make someone else sacrifice a thing...